cmd/restic/cmd_backup.go

```diff
@@ -100,6 +103,21 @@ type BackupOptions struct {
 var backupOptions BackupOptions
 
 func init() {
+	//set FileReadConcurrency to 4 is not set in env
+	filereadconcurrency, err := strconv.Atoi(os.Getenv("RESTIC_FILE_READ_CONCURRENCY"))
+	if err != nil {
+		filereadconcurrency = 2
+	} else if filereadconcurrency < 1 {
+		filereadconcurrency = 1
+	}
+	//set SaveBlobConcurrency to number of procs if not set in env
+	saveblobconcurrency, err := strconv.Atoi(os.Getenv("RESTIC_SAVE_BLOB_CONCURRENCY"))
+	if err != nil {
+		saveblobconcurrency = runtime.NumCPU()
```

Comment:
This duplicates code in the archiver. So either remove it there or get rid of the duplication here.
This also needs guard-rails to prevent setting too high values.